src/styles/main.scss

```
@@ -0,0 +1,103 @@
+$day-size: 100px;
+$transition-time: 0.5s;
+$calendar-width: 7 * ($day-size + 1px);
+$days-of-week: (
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+  sun: 6,
+);
+$calendar-padding: 20px;
+$bg-color: #eee;
+
+body {
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  margin: 0;
+  height: 100vh;
+}
```

why are you using these properties for the `body` tag?

also use the `reset.scss` file to reset the properties
```suggestion
body {
  margin: 0;
}
```
use these properties in classes where you need them